scrape.py

```diff
@@ -9,7 +9,7 @@
 
 from resources.pyterm_colors import pyterm_colors
 
-url = "https://summerofcode.withgoogle.com/archive/2018/organizations/"
+url = "https://summerofcode.withgoogle.com/archive/2019/organizations/"
```

Comment:
Shouldn't this be 2020?

Reply:
The 2020 is not archived yet and https://summerofcode.withgoogle.com/organizations/ is currently the link for 2020

Reply:
Makes sense. The way this scraper was designed earlier was to query organizations that were released in a particular year and search backward. We can probably add support to 2020 once it is archived